packages/contracts/contracts/L2/predeploys/iTssRewardContract.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Rename this file to 'ITssRewardContract.sol'?

Reply:
as L2 predeploys, it start with low case: i